js/main.js

```diff
@@ -228,6 +228,16 @@ function setupThenOpenMainWindow() {
     isAppAlreadyOpen = true;
     getUrlAndCreateMainWindow();
     
+    // Allows a developer to set custom user data path from command line when
+    // launching the app. Mostly used for running automation tests with
+    // multiple instances
+    let customDataArg = getCmdLineArg(process.argv, '--userDataPath=', false);
+    
+    if (customDataArg && customDataArg.split('=').length > 1) {
+        let customDataFolder = customDataArg.split('=')[1];
```

Comment:
@VishwasShashidhar I don't think this would work if the path has '='

ex: --userDataPath="C:\Users\Test_user\Desktop\n=ote.txt"
will result in [ '--userDataPath', '"C:\\Users\Test_user\Desktop\n', 'ote.txt"' ]

Reply:
as confirmed by Daniel, they just pass the "path" of the folder and not the filename itself.

<img width="1128" alt="screen shot 2018-05-07 at 15 43 29" src="https://user-images.githubusercontent.com/5968790/39697114-79c7618c-520d-11e8-9aee-6eeeb3e6d6a1.png">

Reply:
I agree with @KiranNiranjan here. Maybe we could identify the first "=" with a regex and then get a substring after that position.

Please check the screenshot attached here. It's entirely possible that we could name a folder with "=" in it. So, this solution might break for those cases.

<img width="292" alt="screen shot 2018-05-09 at 10 30 11 am" src="https://user-images.githubusercontent.com/27778544/39796450-24b64a9c-5374-11e8-9dd5-ae860a9c2d08.png">